### PR TITLE
Add support for italic text

### PR DESCRIPTION
**Pull request by buztard**

### CODE REVIEW

File: src/window.cpp [hunk 1]
```diff
@@ -689,6 +693,13 @@ void Window::underline(bool underline_state) const
 	(underline_state ? wattron : wattroff)(m_window, A_UNDERLINE);
 }
 
+void Window::italic(bool italic_state) const
+{
+#ifdef A_ITALIC
```
Comment (arybczak) [inline src/window.cpp]: Why is there ifdef?

**Comment by buztard** [comment]
A_ITALIC is a *pretty new* addition to ncurses (late 2013 IIRC), so I added it to make sure it won't break some ancient installations. Of course, a cleaner approach would be to check for the required version of ncurses in configure.ac, but at the time of this PR it was way too late at night to touch any autotools stuff. I'm sorry about that

**Comment by arybczak** [comment]
Can you make it check the version?

**Comment by arybczak** [comment]
No feedback, closing.

**Pull request closed without merging**